src/constants.ts

```diff
@@ -0,0 +1,4 @@
+const KEY_NAME = 'DISCORD_NICKNAME_CHANGED';
+const NAMESPACE_NAME = 'CRON_JOBS_TIMESTAMPS';
```

Comment:
key name is very generic name, are we gonna use this key name anywhere else?

Reply:
Done